eng/scripts/CodeChecks.ps1

```diff
@@ -101,7 +98,11 @@ try {
         if ($LastExitCode -ne 0) {
             $status = git status -s | Out-String
             $status = $status -replace "`n","`n    "
-            LogError "Generated code is not up to date. You may need to run eng\scripts\Update-Snippets.ps1 or sdk\storage\generate.ps1 or eng\scripts\Export-API.ps1"
+            LogError "Generated code is not up to date.`n" + `
+                "You may need to run rebase on the latest master, `n" + `
+                "run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files, `n" + `
```

Comment:
is the idea to recommend to do all of them? or should `or` be added again

Reply:
do one or more depending on what kind of changed PR had